diff --git a/android/app/build.gradle b/android/app/build.gradle
index cce0f24..57ac48e 100644
--- a/android/app/build.gradle
+++ b/android/app/build.gradle
@@ -94,7 +94,23 @@ android {
         versionCode 1
         versionName "0.1.0"
     }
+    splits {
+        abi {
+            reset()
+            enable true
+            universalApk false
+            include "armeabi-v7a", "arm64-v8a"
+        }
+    }
     signingConfigs {
+        release {
+            if (project.hasProperty('MYAPP_UPLOAD_STORE_FILE')) {
+                storeFile file(MYAPP_UPLOAD_STORE_FILE)
+                storePassword MYAPP_UPLOAD_STORE_PASSWORD
+                keyAlias MYAPP_UPLOAD_KEY_ALIAS
+                keyPassword MYAPP_UPLOAD_KEY_PASSWORD
+            }
+        }
         debug {
             storeFile file('debug.keystore')
             storePassword 'android'
@@ -109,7 +125,7 @@ android {
         release {
             // Caution! In production, you need to generate your own keystore file.
             // see https://reactnative.dev/docs/signed-apk-android.
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.release
             shrinkResources (findProperty('android.enableShrinkResourcesInReleaseBuilds')?.toBoolean() ?: false)
             minifyEnabled enableProguardInReleaseBuilds
             proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
diff --git a/android/gradle.properties b/android/gradle.properties
index 7531e9e..4aff03b 100644
--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -28,7 +28,7 @@ android.enablePngCrunchInReleaseBuilds=true
 # Use this property to specify which architecture you want to build.
 # You can also override it from the CLI using
 # ./gradlew <task> -PreactNativeArchitectures=x86_64
-reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
+reactNativeArchitectures=armeabi-v7a,arm64-v8a
 
 # Use this property to enable support to the new architecture.
 # This will allow you to use TurboModules and the Fabric render in
diff --git a/app.config.js b/app.config.js
index ee07b04..a60f6ad 100644
--- a/app.config.js
+++ b/app.config.js
@@ -1,6 +1,6 @@
 module.exports = {
   expo: {
-    name: "Waifu Walls DEV",
+    name: "Waifu Walls",
     slug: "waifuwalls",
     scheme: "waifuwalls",
     version: "0.1.0",
@@ -24,7 +24,7 @@ module.exports = {
           },
         },
       ],
-      package: "com.rks.waifuwalls_dev",
+      package: "com.rks.waifuwalls",
     },
     plugins: [
       "expo-sqlite",
